graph/memstore/quadstore.go

```diff
@@ -120,7 +140,7 @@ func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta, ignoreOpts graph.IgnoreOp
 				err = nil
 			}
 		default:
-			err = errors.New("memstore: invalid action")
+			panic("memstore: still invalid action?")
```

Comment:
This should never happen, but s/still invalid action\?/unexpected invalid action/